erpnext/setup/install.py

```diff
@@ -66,6 +67,16 @@ def create_compact_item_print_custom_field():
 	})
 
 
+def create_print_uom_after_qty_custom_field():
+	create_custom_field('Print Settings', {
```

Comment:
Why custom field and not standard field?

Reply:
How can I do this? Print Settings are part of frappe/frappe, not erpnext. It's the only solution I found in erpnext code. I asked in forum, but nobody helped.